build.gradle

```diff
@@ -54,6 +62,45 @@ subprojects {
     tasks.withType(Test) {
         // Getting javadoc to compile is part of the test suite to ensure we are able to publish our artifacts
         dependsOn project.javadoc
+
+        testLogging {
+            events "passed", "skipped", "failed"
+            exceptionFormat "full"
+            showExceptions true
+            showCauses true
+            showStackTraces true
+            showStandardStreams = true
+        }
+        systemProperty 'disableMemoryLeakTests', 'true'
+
+        // Provide way to exclude particular tests from CLI
+        // e.g. ../gradlew test -PexcludeTests=**/KafkaProtobufConsumerLongTermTest*
+        if (project.hasProperty('excludeTests')) {
+            exclude project.property('excludeTests')
+        }
+
+        useJUnitPlatform()
+        //  Disable parallel test execution, see MIGRATIONS-1666
+        systemProperty 'junit.jupiter.execution.parallel.enabled', 'false'
+        systemProperty 'log4j2.contextSelector', 'org.apache.logging.log4j.core.selector.BasicContextSelector'
+        // Verify assertions in tests
+        jvmArgs '-ea'
+    }
+
+    // Mutually exclusive tests to avoid duplication
+    tasks.named('test') {
+        useJUnitPlatform {
+            excludeTags 'longTest'
+        }
+        jacoco.enabled = true
+    }
+
+    tasks.register('slowTest', Test) {
+        useJUnitPlatform {
+            includeTags 'longTest'
+        }
+        systemProperty 'disableMemoryLeakTests', 'false'
```

Comment:
Can we get rid of this environment variable and tag these tests with `longTest`?

Reply:
[Offline] Yea, its more complex than I first thought.  I'd be OK leaving this enabled when running slowTests all the time

Reply:
False == not specifying at all, i'm removing false and only specifying it as true where needed